Formalisation(shacl)/Core/PiecesShape/Dataset.ttl

```diff
@@ -153,3 +154,26 @@
   ];
 
 sh:targetClass foaf:Agent .
+
+
+:KindShape a sh:NodeShape;
+  sh:property     
+  [ 
+    sh:path vcard:hasEmail ;
+    sh:nodeKind sh:IRI ;
+    dash:viewer dash:LabelViewer ;
+    dash:editor dash:URIEditor ;
```

Comment:
Not sure if `dash:LabelViewer` is the most appropriate here, there will not be a label to resolve for this.

Reply:
changed to dash:viewer dash:URIViewer.
There are other dash:LabelViewer pair with dash:URIEditor, which are working correctly so far. Do we need to change all to dash:URIViewer, or wait and do later?